nav2_costmap_2d/src/clear_costmap_service.cpp

```diff
@@ -88,6 +105,37 @@ void ClearCostmapService::clearExceptRegion(const double reset_distance)
   }
 }
 
+void ClearCostmapService::clearAroundRobot(double window_size_x, double window_size_y)
```

Comment:
The default parameters (0,0) should clear the full costmap. Right now it wouldnt clear anything.

Reply:
Fixed